### DISCUSSION TITLE
Dashboard button for Shutdown/Power OFF rpi

### DISCUSSION THREAD

**Discussion by janreichl** [JaapvanEkris/openrowingmonitor#55] [Q&A] [answered]
Hi 
Great project.
I have ORM(0.9.0) running on a rpi4 and the dashboard data displayed on a webpage with rowing data/actions etc.
Now I have 5 action buttons: Reset, Fullscreen, HRM, FEANT, BLE.

I don't have a button on the dashboard for Shutdown/PowerOFF , how do you get Shutdown/PowerOFF button on dashboard?
Do I need to add in config or how to get the Shutdown/PowerOff button on the dashboard?

**Comment by JaapvanEkris** [comment, marked as answer]
Point your browser to http://your-ip/?mode=kiosk
